### PR TITLE
Added hover effect on navbar items which fixes #68

### LINKED ISSUE #68: 🚀 [New Feature_LP] - Hover effect on navbar
### Description of the enhancement. ✍🏻

Currently, when we hover over the navbar only the opacity of the links changes, this is not that attractive. 

### Describe briefly how you would implement this enhancement.

```bash
Some steps to implement this enhancement-
1. Instead, we can add hover and transition effects on the links to make them more attractive
```

### Screenshots Example (if applicable)

Look at the below videos for more context :

Before :

https://github.com/CatsInTech/GitHub-ReadMe/assets/76463001/b9f805fe-ddb7-4c7b-b47e-708e4bd28010

After :

https://github.com/CatsInTech/GitHub-ReadMe/assets/76463001/e3c5368a-fb20-4b4b-8c9d-8c572d8f4fba

### Additional information

@praveenscience , I'd like to work on this issue.

### Contributor terms

- [X] I agree to follow the Contributing Instructions

### PR DESCRIPTION
**Pull request by mukul314**
## Related Issue(s)

Closes #68 

## Proposed Changes

Added hover effect on the navbar. Now it looks more appealing.

## Type of Change

- [x] New Feature (Landing Page)
- [ ] New Feature (App)
- [ ] Bug (Landing Page)
- [ ] Bug (App)
- [ ] New Component (App)
- [ ] Enhancement
- [ ] Documentation / ReadMe / Workflows
- Other (If not in above options):  

## Your Idea for Level (GSSoC)

- [x] Level 1
- [ ] Level 2
- [ ] Level 3

## Screenshots (if applicable)

Before :

2023-06-03 22_06_00-GitHub ReadMe_ Elevating Your Projects   Profile to New Heights! - Brave

After:

2023-06-03 22_04_45-GitHub ReadMe_ Elevating Your Projects   Profile to New Heights! - Brave

### CODE REVIEW

**Review by praveenscience — CHANGES_REQUESTED** [review]
Use the right PR template, thanks.

**Comment by mukul314** [comment]
> Use the right PR template, thanks.

I edited that, is it okay now @praveenscience ?

File: css/style.css [hunk 1]
```diff
@@ -538,6 +538,11 @@ select.form-control.has-value {
 .bg-transparent ul li a:hover,
 .white-bg.affix ul li a:hover {
   opacity: 1;
+  background-color: #7202bb;
+  color: #ffffff;
+  transition: background-color 0.3s, color 0.3s;
+  border-radius: 5px;
+  padding: 5px 10px;
```
Comment (praveenscience) [inline css/style.css:545]: Can we have these two in the main `a`?

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (mukul314) [reply css/style.css:545]: Okay I'll do that.

Comment (praveenscience) [reply css/style.css:545]: Correct! 😊

File: css/style.css [hunk 2]
```diff
@@ -309,6 +309,7 @@ b {
 
 a {
   color: #3264f5;
+
```
Comment (praveenscience) [inline css/style.css:312]: Remove this line.

File: css/style.css [hunk 3]
```diff
@@ -535,9 +536,17 @@ select.form-control.has-value {
   opacity: 1;
 }
 
+.navbar-nav.ml-auto.menu li a{
```
Comment (praveenscience) [inline css/style.css:539]: Add space after `a`.

File: css/style.css [hunk 4]
```diff
@@ -535,9 +536,17 @@ select.form-control.has-value {
...
+  border-radius: 5px;
+  padding: 5px 10px; 
+}
+
 .bg-transparent ul li a:hover,
 .white-bg.affix ul li a:hover {
   opacity: 1;
+  background-color: #7202bb;
+  color: #ffffff;
```
Comment (praveenscience) [inline css/style.css:548]: Make it `#fff`

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**